cumulus/test/runtime/src/lib.rs

```diff
@@ -98,21 +102,27 @@ impl_opaque_keys! {
 /// The para-id used in this runtime.
 pub const PARACHAIN_ID: u32 = 100;
 
-#[cfg(not(feature = "elastic-scaling"))]
+#[cfg(not(any(feature = "elastic-scaling", feature = "elastic-scaling-500ms")))]
 const UNINCLUDED_SEGMENT_CAPACITY: u32 = 4;
-#[cfg(not(feature = "elastic-scaling"))]
+#[cfg(not(any(feature = "elastic-scaling", feature = "elastic-scaling-500ms")))]
 const BLOCK_PROCESSING_VELOCITY: u32 = 1;
+#[cfg(not(any(feature = "elastic-scaling", feature = "elastic-scaling-500ms")))]
+pub const MILLISECS_PER_BLOCK: u64 = 6000;
 
-#[cfg(feature = "elastic-scaling")]
+#[cfg(all(feature = "elastic-scaling", not(feature = "elastic-scaling-500ms")))]
 const UNINCLUDED_SEGMENT_CAPACITY: u32 = 7;
-#[cfg(feature = "elastic-scaling")]
+#[cfg(all(feature = "elastic-scaling", not(feature = "elastic-scaling-500ms")))]
 const BLOCK_PROCESSING_VELOCITY: u32 = 4;
-
-#[cfg(not(feature = "elastic-scaling"))]
-pub const MILLISECS_PER_BLOCK: u64 = 6000;
-#[cfg(feature = "elastic-scaling")]
+#[cfg(all(feature = "elastic-scaling", not(feature = "elastic-scaling-500ms")))]
 pub const MILLISECS_PER_BLOCK: u64 = 2000;
 
+#[cfg(feature = "elastic-scaling-500ms")]
+const UNINCLUDED_SEGMENT_CAPACITY: u32 = 30;
+#[cfg(feature = "elastic-scaling-500ms")]
+const BLOCK_PROCESSING_VELOCITY: u32 = 12;
+#[cfg(feature = "elastic-scaling-500ms")]
+pub const MILLISECS_PER_BLOCK: u64 = 500;
```

Comment:
Can we just define `MILLISECS_PER_BLOCK` and then compute the `BLOCK_PROCESSING_VELOCITY` and `UNINCLUDED_SEGMENT_CAPACITY`  based on it ? It would also reduce the amount cluttering due to feature flags.

Reply:
good idea, done